test/test_oneview_server_profile_facts.py

```diff
@@ -234,6 +226,31 @@ def test_should_get_all_server_profiles_with_options(self):
                            }
         )
 
+    def test_should_get_server_profiles_with_invalid_options(self):
```

Comment:
Wouldn't be better something like 'test_should_get_server_profiles_with_profile_ports_option' ?